content/en/docs/concepts/scheduling-eviction/node-pressure-eviction.md

```diff
@@ -347,7 +345,7 @@ Consider the following scenario:
 For this to work, the kubelet is launched as follows:
 
 ```
---eviction-hard=memory.available<500Mi
+--eviction-hard={"memory.available": "500Mi"}
```

Comment:
I'm not sure this would work. Shouldn't the value for the flag in the command be something like `--eviction-hard=memory-available=500Mi`?